modularframework-spigot/build.gradle

```diff
@@ -0,0 +1,38 @@
+plugins {
+//    id "tech.ferus.gradle.spigotgradle" version "1.0.5"
```

Comment:
This is because we're not making it a plugin yet until we figure out how we're going to do that for Spigot and Sponge.